db-service/lib/cqn4sql.js

```diff
@@ -910,7 +912,11 @@ function cqn4sql(originalQuery, model) {
           elements[c.as || c.ref.at(-1)] = c.element
         })
         return res
-      })
+      }).filter(c => c !== null)
```

Comment:
can `c` be anything `falsy`, but not `null` ?

Reply:
no, either its an object representing a column, or it is null (as a result of a wildcard expand). :)

Reply:
Great then we can simplify the check like: 

```suggestion
      }).filter(c => c)
```